Lombiq.NodeJs.Extensions/package.json

```diff
@@ -15,8 +15,9 @@
     "stylelint-config-standard-scss": "^3.0.0"
   },
   "scripts": {
-    "dotnet-prebuild": "pnpm build:styles",
-    "dotnet-postclean": "pnpm clean:styles",
+    "build": "pnpm build:styles",
+    "compile": "pnpm compile:styles",
+    "clean": "pnpm clean:styles",
```

Comment:
Why where these changes necessary?

Reply:
Since in the `.props` file I call either `build` or `compile`, depending on the value of `$(NuGetBuild)`, it seemed a good idea to use these short script names instead of `dotnet-prebuild` and `dotnet-postclean`. Choosing `build` over `build:styles` also keeps it open to extension by JS compilation in the future.